### PR TITLE
Strings for changelog

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

* stubs for changelog strings
* adds context which was recently added on crowdin

### Reviewer guidance

are these correct? should they already have existed in the codebase somewhere?

### References

based on:

* https://www.figma.com/file/eJd6xQiNgebzzRWn2XDDS92R/0.13-content-import?node-id=222%3A0
* https://docs.google.com/document/d/1jivSlr5MidAmbBJBkHLps9f3wSR5hLvGmTmCihIkdgI/edit

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/device/assets/src/views/ManageContentPage/channelUpdateStrings.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { createTranslator } from 'kolibri.utils.i18n';
+
+export default createTranslator('ChannelUpdateStrings', {
+  notAvailableFromDrives: 'This channel was not found on any attached drives',
+  notAvailableFromNetwork: 'This channel was not found on other instances of Kolibri',
+  notAvailableFromStudio: 'This channel was not found on Kolibri Studio',
+  newVersionAvailableHeader: "Version {version} of '{channel}' is available",
+  currentVersionLabel: 'You are currently on version {version}',
```
Comment (jonboiser) [inline kolibri/plugins/device/assets/src/views/ManageContentPage/channelUpdateStrings.js]: These messages are in NewChannelVersionPage, but have different keys and slightly different punctuation. e.g. `New resources available:` (has a colon at the end).

**Comment by indirectlylit** [comment]
updated to incorporate stubbed out components

File: kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue [hunk 2]
```diff
@@ -94,17 +94,39 @@
...
+      resourcesToBeUpdated: {
+        message: 'Resources to be updated',
+        context: 'Label associated with the number of resources would be updated',
+      },
       updateChannelAction: 'Update channel',
       versionNumberHeader: 'Version {version}',
       updateConfirmationQuestion: `Are you sure you want to update '{channelName}' to version {version}?`,
-      channelIsIncomplete:
-        'This channel source is incomplete. It has {available} of {total} resources',
+      channelIsIncomplete: {
```
Comment (jonboiser) [inline kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue:124]: Revert this change, we're using the original version on the left. `'This channel source is incomplete. It has {available} of {total} resources',`

File: kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue [hunk 3]
```diff
@@ -94,17 +94,39 @@
...
       versionIsAvailable: `Version {nextVersion} of '{channelName}' is available`,
       youAreCurrentlyOnVersion: 'You are currently on version {currentVersion}',
-      versionChangesHeader: 'Changes if you update from version {currentVersion} to {nextVersion}',
-      resourcesAvailableForImport: 'New resources available:',
-      resourcesToBeDeleted: 'Resources that will be deleted:',
-      resourcesToBeDeletedTooltip:
-        'When you update this channel, some resources will be deleted. This may affect lessons or quizzes that are using the deleted resources.',
-      resourcesToBeUpdated: 'Resources to be updated:',
+      versionChangesHeader: {
+        message: 'Changes if you choose to update from version {oldVersion} to {newVersion}:',
```
Comment (jonboiser) [inline kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue]: Revert this change, we're going to use the version on the left.

File: kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue (hunk 2 above)
Comment (indirectlylit) [reply kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue:124]: fixed variable names. user-facing text didn't change - just added context

Comment (indirectlylit) [reply kolibri/plugins/device/assets/src/views/ManageContentPage/NewChannelVersionPage.vue:124]: fixed variable names. kept new phrasing for now - let's discuss in Slack

**Review by jonboiser — APPROVED** [review]

**Comment by rtibbles** [comment]
Build breakage is debian only, merging.
